sklearn/feature_selection/tests/test_multivariate_select.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```

Comment:
I think that these tests should just be added to the `test_feature_select.py` file

Reply:
add this line: https://github.com/srecupe2/scikit-learn/blob/e42fb227b891593520525cf6cb6302f0edfc12eb/sklearn/feature_selection/_univariate_selection.py#L1

Reply:
https://github.com/srecupe2/scikit-learn/pull/4/commits/184e5432f44d112873543cc2c7e0ae22e2ebb3f7